UnityProject/Assets/Scripts/Player/Mind.cs

```diff
@@ -19,7 +19,14 @@
 /// </summary>
 public class Mind : MonoBehaviour
 {
+	public GameObject PossessingObject;
+	public IPlayerPossessble IPlayerPossessble;
```

Comment:
```suggestion
	private GameObject possessedObject;
	private IPlayerPossessble possessed;
```

Reply:
I think you'd want these being public so you can access them through the mind, 

For example if you want to know where the player is you would do mind.possessedObject.position?

Reply:
Public getter private setter?

Reply:
ok